### PR TITLE
Add console port to container execution for AMQP dev services

### PR DESCRIPTION
**Pull request by hguerrero**
This change updates the devservice container for amqp to enable the console available in artemiscloud image.

Follow up to [#22976](https://github.com/quarkusio/quarkus/issues/22976)

### CODE REVIEW

File: extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesProcessor.java [hunk 1]
```diff
@@ -277,14 +278,15 @@ private ArtemisContainer(DockerImageName dockerImageName, String extra, int fixe
...
-            withExposedPorts(AMQP_PORT);
+            withExposedPorts(AMQP_PORT, AMQP_CONSOLE_PORT);
             withEnv("AMQ_USER", DEFAULT_USER);
             withEnv("AMQ_PASSWORD", DEFAULT_PASSWORD);
             withEnv("AMQ_EXTRA_ARGS", extra);
             if (serviceName != null) { // Only adds the label in dev mode.
                 withLabel(DEV_SERVICE_LABEL, serviceName);
             }
-            if (dockerImageName.getRepository().endsWith("artemiscloud/activemq-artemis-broker")) {
+            if (dockerImageName.getRepository().startsWith("artemiscloud/activemq-artemis-broker")) {
```
Comment (gsmet) [inline extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesProcessor.java:288]: Why this change?

Comment (hguerrero) [reply extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesProcessor.java:288]: Good point. This remains when we try to use other broker images based on artemiscloud. I will remove it.

**Comment by hguerrero** [comment]
@gsmet rolledback the repository validation and updated to the latest container image.

**Review by cescoffier — APPROVED** [review]
Thanks @hguerrero

File: extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesBuildTimeConfig.java [hunk 2]
```diff
@@ -32,13 +32,13 @@ public class AmqpDevServicesBuildTimeConfig {
...
      */
-    @ConfigItem(defaultValue = "quay.io/artemiscloud/activemq-artemis-broker:0.1.2")
+    @ConfigItem(defaultValue = "quay.io/artemiscloud/activemq-artemis-broker:1.0.11")
     public String imageName;
 
     /**
      * The value of the {@code AMQ_EXTRA_ARGS} environment variable to pass to the container.
      */
-    @ConfigItem(defaultValue = "--no-autotune --mapped --no-fsync")
+    @ConfigItem(defaultValue = "--no-autotune --mapped --no-fsync --relax-jolokia --http-host 0.0.0.0")
```
Comment (cescoffier) [inline extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesBuildTimeConfig.java:41]: For my own curiosity: what does relax-jolokia do?

Comment (hguerrero) [reply extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesBuildTimeConfig.java:41]: By default Artemis restricts the list or allowed origins to access the jolokia console

```xml
<allow-origin>*://0.0.0.0*</allow-origin>
```

This flag removes the restriction to allow external connections to the container.
